common/src/main/java/dev/latvian/kubejs/server/TagEventJS.java

```diff
@@ -26,6 +27,10 @@
 /**
  * @author LatvianModder
  */
+@KubeJSEvent(
+		server = { "<ItemTypeName>.tags" },
+		client = { "<ItemTypeName>.tags" }
```

Comment:
Tag event doesnt get fired on client. Also I would rather list the 4 basic tags - item, block, entity_type, fluid